src/pages/FormPageHoc.tsx

```diff
@@ -24,13 +24,21 @@ export type ValidateFormParams<P> = {
   formProps: P;
 };
 
-export const FormPageHoC = <P extends Record<string, unknown>>(
+interface FormPageHoCProps<P extends Record<string, unknown>> {
   FormComponent: (
     props: P & { errors: ValidationErrorItem[] | undefined }
-  ) => JSX.Element,
-  formProps: P,
-  validatePage?: (params: ValidateFormParams<P>) => ValidationErrorItem[]
-) => {
+  ) => JSX.Element;
+  formProps: P;
+  validatePage?: (params: ValidateFormParams<P>) => ValidationErrorItem[];
+  twoThirdsWidth?: boolean;
+}
+
+export const FormPageHoC = <P extends Record<string, unknown>>({
```

Comment:
just realised we used "HoC" in the component and interface and "Hoc" (lowercase c) for the file name. Shall we make these 2 consistent? I think I personally prefer "Hoc" with lowercase c

Reply:
I think it probably should have been "HoC" but I don't have a strong opinion on it and changing file name casing in Window is a pain so I'll go lower case 😆